src/components/CallToAction.css

```diff
@@ -0,0 +1,31 @@
+/* * {
+  border: 1px solid red;
+} */
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;500&display=swap");
+
+.main-cta-wrapper {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  width: 360px;
```

Comment:
Maybe we should implement rem values to make it also more responsive

Reply:
I think this a great idea 👍 

We can setup a font-size on the html with media queries and start specificying sizes in rems @sjaak1992 

(In the case of the width of this card, it's 360px on the desktop and 360px on the mobile design, so maybe in this case the px's are okay)

Reply:
Yes perfect!